src/sentry/static/sentry/app/views/organizationDiscover/result/index.jsx

```diff
@@ -74,7 +77,9 @@ export default class Result extends React.Component {
       return {
         isEditMode,
         savedQueryName: isEditMode
-          ? savedQuery ? savedQuery.name : generateQueryName()
+          ? savedQuery
```

Comment:
What version of prettier do you have?

Reply:
it was like 1.8, now realizing it's newer than the one we use.